packages/nimble-components/src/anchor/tests/anchor-matrix.stories.ts

```diff
@@ -85,3 +85,12 @@ export const textWrapping: StoryFn = createStory(
     </p>
     `
 );
+
+export const inlineSpacing: StoryFn = createStory(
+    html`<style>
+        .inlineLink {
```

Comment:
we generally use kebab case for class names. If we need to capture that in the CSS docs or something we should. i.e. `.inline-link`

Reply:
Added a note to our coding conventions doc. That seemed like a better place than the CSS docs, because the CSS only references class names--it does not define them.